### DISCUSSION TITLE
Moderation?

### DISCUSSION THREAD

**Discussion by danielgallab** [FlagClicked/FlagClicked#20] [Ideas]
Should we have moderation? Or just a simple "Report" Button where with enough reports it goes down? Maybe both sorta?

How will we do this system? How do we make sure no one else can become a mod with code (like you can do with scratch)?

So many questions that we need some details for.

**Comment by retronbv** [comment]
> How do we make sure no one else can become a mod with code (like you can do with scratch)?

\>:)

**Comment by retronbv** [comment]
I think we should have all posts flagged for moderation BEFORE they are uploaded. Have them in a kind of “limbo” state

**Comment by markverb1** [comment]
> Should we have moderation? Or just a simple "Report" Button where with enough reports it goes down? Maybe both sorta?
> 
> How will we do this system? How do we make sure no one else can become a mod with code (like you can do with scratch)?
> 
> So many questions that we need some details for.

We will probably need to verify if the tutorial is good enough for the website so I agree with retronbv's idea.
